docs/vc_patch.md

```diff
@@ -0,0 +1,113 @@
+# Nintendo 2DS/3DS Virtual Console Patch
+
+The Nintendo Virtual Console for the 2DS and 3DS can play Game Boy Color ROMs bundled into a `.cia` (CTR Importable Archive) file. Creating a `.cia` file for a `.gbc` ROM file is outside the scope of this project.
+
+An optional file bundled in the `.cia` archive is a `.patch` file with the same name as the `.gbc` file, used to determine when and under what conditions to enable certain Virtual Console functionalities, such as the Wireless Link capabilities. It also allows arbitrary content in the ROM to be modified, such as for disabling the Game Boy Printer features of Pokémon Crystal.
+
+
+## Build pokecrystal11_vc.patch
+
+To build **pokecrystal11_vc.patch**:
+
+```bash
+make crystal11_vc
+```
+
+This will also create two ROM files, **pokecrystal11.gbc** and **pokecrystal11_vc.gbc**. The pokecrystal11_vc.gbc file has the patches already applied to it. Do *not* use this file; rename pokecrystal11_vc.patch to **pokecrystal11.patch** and bundle it together with pokecrystal11.gbc in a `.cia` file.
```

Comment:
"Do *not* use this file; the pokecrystal11_vc.patch file will apply its changes on top of the original pokecrystal11.gbc file. These two files should be renamed to match, and always be used together."

Reply:
I wonder if we should fix this within the Makefile though. Why not output pokecrystal11.patch instead of 11_vc.patch?

Reply:
I'll try that.